src/PlayerActor.cpp

```diff
@@ -0,0 +1,33 @@
+#include "PlayerActor.h"
+
+
+PlayerActor::PlayerActor(){
```

Comment:
implement init()

Reply:
basically init should set the default values. Since we have a init in the Actor.h, we should implement it here in the concrete class.